### ISSUE TITLE
Repetitiveness in real life software documentation

### ISSUE THREAD

**Issue by mrsumitbd** [marouen-lamiri/SOEN-6611#21]
## Description
Our goal is to determine whether any repetitiveness can be found in real life software documentation.
## User Story Attributes
Attribute | Value
------------- | -------------
Risk  | Medium
USP  | 5
Priority | High

## Task Breakdown
- [x] RawData: Javadoc
- [x] AttributesExtraction: Cross-entropy of the documents
- [x] StoreLink: 
- [x] Model: Pattern
- [x] Reporting: cross-entropy vs n-gram (x axis = n-gram, y axis = cross entropy)

Assigned to: @arashiqurrahman and @nafiz351

**Comment by mrsumitbd** [comment]
![javadoc](https://cloud.githubusercontent.com/assets/8269579/15234204/4a75505e-187d-11e6-9b5a-2a4a465de6ab.png)

**Comment by mrsumitbd** [comment]
We found exact same pattern in the cross-entropy of JavaDoc.
